File: unity/Assets/Hallway/Constructor/RPG/logotypeJump.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.logotypeJump) {
		scr1.Achievement("Прыжок в логотип!\n+10 очков!", 10);
		scr1.RPG.logotypeJump = true;
	}
}

File: unity/Assets/Hallway/Constructor/RPG/logotypeJump.cs
using UnityEngine;

public class logotypeJump : MonoBehaviour
{
    public GameObject Bootstrap;
    void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.logotypeJump)
        {
//            rpgParser.Achievement("Прыжок в логотип!\n+10 очков!", 10);
//            rpgParser.RPG.logotypeJump = true;
        }
    }
}
